src/EditorServicesCommandSuite/CodeGeneration/Refactors/CommandSplatRefactorSettings.cs

```diff
@@ -12,5 +12,17 @@ internal class CommandSplatRefactorSettings : RefactorConfiguration
         [Parameter]
         [DefaultFromSettingAttribute("CommandSplatRefactor.NewLineAfterHashtable", Default = "$true")]
         public SwitchParameter NewLineAfterHashtable { get; set; }
+
+        [Parameter]
+        [DefaultFromSettingAttribute("CommandSplatRefactor.AllParameters", Default = "$false")]
```

Comment:
I did not notice I accidentally included `Attribute` on the end of these attributes, whoops!

Should be `DefaultFromSetting`, my bad :)

Reply:
That's right, it's the latter part that applies here though.  The idea is to include the suffix in the class definition, but when using the attribute you can omit it.

Like `Parameter` above it, the actual class name is `ParameterAttribute`, but when using it as an attribute you can omit the suffix.

Reply:
Aaah that's what i missed there :) i've fixed that for ya ;-)